<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>añadir producto</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="1" column="1">
    <widget class="QSpinBox" name="quantity_spinbox"/>
   </item>
   <item row="2" column="0" colspan="2">
    <widget class="QPushButton" name="add_product">
     <property name="text">
      <string>Añadir producto</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Cantidad:</string>
     </property>
    </widget>
   </item>
   <item row="0" column="0" colspan="2">
    <widget class="QTableWidget" name="products_table"/>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
